src/lib/rover_control/rovercontrol_params.c

```diff
@@ -0,0 +1,255 @@
+ *
+ *   Copyright (c) 2025 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file rovercontrol_params.c
+ *
+ * Parameters defined by the rover control lib.
+ */
+
+/**
+ * Yaw stick deadzone
+ *
+ * Percentage of stick input range that will be interpreted as zero around the stick centered value.
+ *
+ * @min 0
+ * @max 1
+ * @increment 0.01
+ * @decimal 2
+ * @group Rover Rate Control
+ */
+PARAM_DEFINE_FLOAT(RO_YAW_STICK_DZ, 0.1f);
+
+/**
+ * Yaw rate measurement threshold
+ *
+ * The minimum threshold for the yaw rate measurement not to be interpreted as zero.
+ *
+ * @unit deg/s
+ * @min 0
+ * @max 100
+ * @increment 0.01
+ * @decimal 2
+ * @group Rover Rate Control
+ */
+PARAM_DEFINE_FLOAT(RO_YAW_RATE_TH, 3.f);
+
+/**
+ * Proportional gain for closed loop yaw rate controller
+ *
+ * @min 0
+ * @max 100
+ * @increment 0.01
+ * @decimal 3
+ * @group Rover Rate Control
+ */
+PARAM_DEFINE_FLOAT(RO_YAW_RATE_P, 0.f);
+
+/**
+ * Integral gain for closed loop yaw rate controller
+ *
+ * @min 0
+ * @max 100
+ * @increment 0.01
+ * @decimal 3
+ * @group Rover Rate Control
+ */
+PARAM_DEFINE_FLOAT(RO_YAW_RATE_I, 0.f);
+
+/**
+ * Yaw rate limit
+ *
+ * Used to cap yaw rate setpoints and map controller inputs to yaw rate setpoints
+ * in Acro, Stabilized and Position mode.
+ *
+ * @unit deg/s
+ * @min 0
+ * @max 10000
+ * @increment 0.01
+ * @decimal 3
```

Comment:
```suggestion
 * @decimal 2
```
It was like that in the previous implementation.